algorithms/exact_change.js

```diff
@@ -0,0 +1,84 @@
+/*
+https://www.freecodecamp.com/challenges/exact-change
+
+Design a cash register drawer function checkCashRegister() that accepts purchase
+ price as the first argument (price), payment as the second argument (cash), and
+  cash-in-drawer (cid) as the third argument.
+
+cid is a 2D array listing available currency.
+
+Return the string "Insufficient Funds" if cash-in-drawer is less than the change
+ due. Return the string "Closed" if cash-in-drawer is equal to the change due.
+
+Otherwise, return change in coin and bills, sorted in highest to lowest order.
+
+
+*/
+
+
+// Create an object which hold the denominations and their values
+var denom = [
```

Comment:
More explicit naming could be helpful, e.g., `nameToDenomination`.

Also, why not have an object that maps the string to the value itself, e.g., 

```javascript

const nameToDenomination = {
  'ONE HUNDRED': 100.00,
  ...
}
```